extension/resources/walkthrough/more-resources.md

```diff
@@ -0,0 +1,33 @@
+# More Resources
+
+- Get started with a [hands-on tour] of DVC's major features. Refer to the the
+  [User Guide] for deeper knowledge on the approach and mechanics of DVC.
+
+  > **Note**: Some features and options of DVC are only accessible via
+  > command-line.
```

Comment:
I renamed this in a Git commit before editing. Idk why GH is not picking up the move so the diff is shown, sorry!

<img width="1132" alt="image" src="https://user-images.githubusercontent.com/1477535/176322580-b700fbfb-429a-491b-b182-3f25fea9ee0a.png">